src/battle_script_commands.c

```diff
@@ -3206,7 +3292,7 @@ void SetMoveEffect(u32 battler, u32 effectBattler, bool32 primary, bool32 certai
         if (NoAliveMonsForEitherParty()
           || ChangeStatBuffs(
                 effectBattler,
-                SET_STAT_BUFF_VALUE(1),
+                1,
                 gBattleScripting.moveEffect - MOVE_EFFECT_ATK_PLUS_1 + 1,
```

Comment:
Rereading this code I got a little confused about what operations were being done. I think it might be a little more clear as (and similar for the others below):
```suggestion
                STAT_ATK + (gBattleScripting.moveEffect - MOVE_EFFECT_ATK_PLUS_1),
```